apps/src/gamelab/GameLab.js

```diff
@@ -443,26 +456,42 @@ GameLab.prototype.init = function(config) {
  * @param {Object} expoOpts
  */
 GameLab.prototype.exportApp = async function(expoOpts) {
```

Comment:
Is there anything here that can be combined between applab and gamelab?

Reply:
Not really. Most has been factored into `expoInteractWithApk()`

Gamelab has a different way of grabbing studio app (`this.studioApp_`)

It also needs to do the special `whenAnimationsAreReady()` section.

Reply:
👍